src/index.ts

```diff
@@ -18,16 +27,44 @@ const formatError = winston.format(info => {
     return info;
 });
 
+/* A custom format that is used to include tenantId. */
+const formatTenantId = (parameters: ParameterConfig | undefined) => {
+    return winston.format(info => {
+        info.tenantId = "";
```

Comment:
Remove this as well. If we get tenantId from the context then use it, else make no changes.